metric-anomaly-data-model/src/main/avro/MetricAnomalyViolation.avdl

```diff
@@ -0,0 +1,9 @@
+@namespace("org.hypertrace.alert.engine.metric.anomaly.datamodel")
+protocol MetricAnomalyViolationProtocol {
+  record MetricAnomalyViolation {
```

Comment:
Should we rename it as MetricAnomalyActionEvent or MetricAnomalyNotificationEvent?